package.json

```diff
@@ -31,6 +31,7 @@
   "devDependencies": {
     "coveralls": "^2.10.0",
     "format-stack": "4.0.0",
+    "get-port": "^1.0.0",
```

Comment:
get-port is a hack with an inherent race condition. It is possible in the narrow window between when get-port closes a socket and when you listen on it again, that it will have been given to another process.

listen(0) is the only reliable way to get a socket from the operating system.